features/other_a/gift.feature

```diff
@@ -391,3 +391,60 @@ Feature: Create Gifts
       And I should not see "by gifter for giftee1"
     When I view the work "Rude Gift"
     Then I should not see "For giftee1."
+
+  Scenario: Translated email is sent when a regular work is gifted
+    Given a locale with translated emails
+      And the user "giftee1" enables translated emails
+      And all emails have been delivered
+    When I give the work to "giftee1"
+      And I press "Post"
+      And I set up the draft "GiftStory2" as a gift to "giftee2"
+      And I press "Post"
+    Then "giftee1" should be emailed
+      And the email should have "\[AO3\] A gift work for you" in the subject
+      And the email to "giftee1" should be translated
+    Then "giftee2" should be emailed
+      And the email should have "\[AO3\] A gift work for you" in the subject
```

Comment:
Can you remove the `[AO3]` from the subject checks? It's a bit fragile if someone changes this abbreviation in the configuration in their development install.